RecoLocalTracker/Phase2TrackerRecHits/python/Phase2StripCPEESProducer_cfi.py

```diff
@@ -2,5 +2,8 @@
 
 phase2StripCPEESProducer = cms.ESProducer("Phase2StripCPEESProducer",
   ComponentType = cms.string('Phase2StripCPE'),
-  parameters    = cms.PSet()
+  parameters    = cms.PSet(
+    LorentzAngle_DB = cms.bool(False),			
+    TanLorentzAnglePerTesla = cms.double(0.07)
```

Comment:
Yes, I agree with @lowette. Changing names can propagate problems in different part of the code. In this moment we have just one CPE for both with a name that is for sure nor descriptive either optimal, but I would suggest to change it when we will have both CPE for P and S separately..